addons/io_scene_gltf2/__init__.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import sys
```

Comment:
This should go inside of the function where you need it, not at the top-level (see #333).